app/views/layouts/_header.html.erb

```diff
@@ -17,9 +17,10 @@
     <div class="collapse navbar-collapse" id="header-navbar-collapse">
       <form class="form-inline mr-3" id="searchform" autocomplete="off">
         <div class="input-group">
+          <label id="searchLabel" for="searchform_input">
           <input type="text" id="searchform_input" class="form-control search-query typeahead" role="search" qryType="tags" placeholder="<%= t('layout._header.search') %>"  value="<%= params[:query] %>" required>
```

Comment:
Instead of blank `label`, `aria-label` property would be better, right? :)

Reply:
Hey @VladimirMikulic Instead of aria-label aria-role search is already present here :sweat_smile: so I didn't add aria-label, should I change it? Thanks a lot for the review :tada:   

![image](https://user-images.githubusercontent.com/33183263/83191374-855b5680-a151-11ea-8d89-2296b53e984d.png)

Reply:
I didn't see `aria-role`. My mistake, `aria-roles` is fine, no need to change that.